scripts/scripts.js

```diff
@@ -141,6 +142,9 @@ export function decorateMain(main) {
  * @param {Element} doc The container element
  */
 async function loadEager(doc) {
+  loadScript('https://js.arcgis.com/calcite-components/1.8.0/calcite.esm.js');
```

Comment:
anything we would need from these scripts in sequence?

We could await them even in parallel

Reply:
We're not even awaiting them here, just starting their load early without blocking.